src/App/Documentation/components/Card/index.js

```diff
@@ -7,29 +7,25 @@ import MediaObject from "@/MediaObject";
 
 const textArr = ["This is a lot of text", "With some more text", "And then even some more", "Is it really possible to have this much text in one card?", "Yes!"];
 
-const DefaultBehaviour = () => (
+const Overview = () => (
     <>
-        <h2 id="default-behaviour">Default behaviour</h2>
+        <h2 id="overview">Overview</h2>
         <p>
-            Wrap indiviual cards in the <Property value=".cards" /> class to display them lined up next to each other.
-            When the window reaches <Link to="/docs/core/breakpoints">breakpoint</Link> sm the cards will switch to mobile view and take up one row each and stretch to fill the width of the container.
-            To decide how your cards will look use <Property value=".card-default" />, <Property value=".card-brand" />, or <Property value=".card-light" />
+            We offer three different card variants; <Property value=".card-primary" />, <Property value=".card-secondary" /> and <Property value=".card-plain" />.
         </p>
         <ComponentPreview language="html" showCasePanel codeFigure>
-            <div className="cards">
+            <div className="row">
```

Comment:
Do we need this row?

Reply:
Nope, must have missed it when removing the grid.